### PR TITLE
[TR] PIM-5040: Cannot change the unit of a metric

### PR DESCRIPTION
**Pull request by solivier**
| Q                 | A
| ----------------- | ---
| Specs             | Yes
| Behats            | Yes
| Blue CI           | Running
| Changelog updated | Yes
| Review and 2 GTM  | No

### CODE REVIEW

**Comment by sumbobyboys** [comment]
GTM

File: features/product/edit_product.feature [hunk 1]
```diff
@@ -22,6 +25,7 @@ Feature: Edit a product
       | sandal  | other_name  | My awesome sandals                   | en_US  | ecommerce |
       | sandal  | other_name  | My awesome sandals for mobile        | en_US  | mobile    |
       | sandal  | name        | My sandals name                      |        |           |
+      | sandal  | length      | 29 CENTIMETER                        |        |           |
```
Comment (sumbobyboys) [inline features/product/edit_product.feature]: Prétentieux !

File: features/product/edit_product.feature [hunk 2]
```diff
@@ -8,10 +8,13 @@ Feature: Edit a product
     Given a "default" catalog configuration
     And I add the "english" locale to the "mobile" channel
     And the following attributes:
-      | code        | type     | localizable | wysiwyg_enabled | label       | scopable |
-      | description | textarea | yes         | yes             | Description | yes      |
-      | name        | text     | no          |                 | Name        | no       |
-      | other_name  | text     | yes         |                 | Other Name  | yes      |
+      | code        | type       | localizable | wysiwyg_enabled | label       | scopable |
```
Comment (skeleton) [inline features/product/edit_product.feature]: extra space on type column

**Comment by skeleton** [comment]
tiny comment then GTM

**Comment by jjanvier** [comment]
GTM

**Pull request merged**